test/client_test.dart

```diff
@@ -52,12 +52,14 @@ void main() {
   });
 
   test('Get user', () async {
+    await client.signIn(email: email, password: password);
```

Comment:
not sure why you have to call signIn multiple times. client should keeps the session.

Reply:
Looks like you reviewed an outdated version of that file. Part of the test failures were because the client was getting reinitialized on every test, I changed the setUp to setUpAll to fix that.